packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -441,11 +441,12 @@ class AndroidStudio implements Comparable<AndroidStudio> {
       return;
     }
 
+    final String javaDirName = version != null && version.major < 2022 ? 'jre' : 'jbr';
```

Comment:
@crasowas I was specifically curious if there was documentation that this is the correct release where the change happened? In reviewing https://b.corp.google.com/issues/236055498, it sounds like the C4 release was where they change happened, but could not find what version this was.

Reply:
@christopherfujino According to the context, the C4 release may refer to Android Studio 2022.1.1 Canary 4. I tried to verify this guess. Unfortunately, [Android Studio Release Updates: Android Studio Electric Eel Canary 4 now available](https://androidstudio.googleblog.com/2022/06/android-studio-electric-eel-canary-4.html) has no relevant description. However, I can confirm that Android Studio 2022.1.1 Stable, Android Studio 2022.2.1 Preview and Android Studio 2022.3.1 Preview have all changed the embedded JDK location.